content/getting-started/connect-the-pim-4x/step-2.md

```diff
@@ -10,7 +10,7 @@ Indeed, we have an App Store on which our dear ecosystem publishes its own tools
 
 So, we definitely recommend you to take a look at it and try to find a connector that can suit your use case. It will save you a lot of time!
 
-1. Navigate to the [marketplace](https://marketplace.akeneo.com/extensions?edition=all&version=all&api_use=1&sort=date) website.  
+1. Navigate to the [marketplace](https://apps.akeneo.com/extensions?edition=all&version=all&api_use=1&sort=date) website.  
```

Comment:
Are we going to change text occurrences of "marketplace" later?

Reply:
Nope this one has been forgot.
Fixed.